app/views/search/_global_user_role_link.html.haml

```diff
@@ -0,0 +1 @@
+= link_to user.name_last_comma_first, edit_global_user_role_path(user.id)
```

Comment:
Now I see. Inside `user_search_results.html.haml` it does a dynamic partial render (which is probably not super safe, but probably not worth fixing right now...).